internal/pkg/skuba/kubernetes/node_version.go

```diff
@@ -34,6 +34,7 @@ var (
 )
 
 type NodeVersionInfo struct {
+	Node                     *v1.Node
```

Comment:
if we have the whole Node object then `Nodename string` is redundant right?

Reply:
Not necessarily, it contains data fetched from pod versions (apiserver, controller manager, scheduler, etcd) and other metadata fetched from the node for convenience purposes. Some of them could certainly be removed like `Nodename` (accessible through `.Node.ObjectMeta.Name`, but honestly, I think it's fine if we keep it).